include/Util/Transform.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef UTIL_TRANSFORM_HPP
+#define UTIL_TRANSFORM_HPP
+
+#include <glm/glm.hpp>
+#include <glm/gtc/matrix_transform.hpp>
+
+namespace Util {
+class Transform {
+public:
+    Transform() : m_mat3(glm::mat3(1.0f)) {};
+    Transform(glm::mat3 mat3) : m_mat3(mat3) {};
+
+    Transform Rotate(float degrees) const { return RotateByRadians(glm::radians(degrees)); }
+    Transform RotateByRadians(float radians) const;
+
+    Transform Translate(const glm::vec2& translation) const;
+    Transform Translate(float x, float y) const { return Translate(glm::vec2(x,y)); }
+
+    Transform Scale(const glm::vec2& scale) const;
+    Transform Scale(float x, float y) const { return Scale(glm::vec2(x,y)); }
```

Comment:
I don't think overloading here is necessary, if users wanted to they can write their own helper functions